lang/main.json

```diff
@@ -617,7 +617,7 @@
         "accessibilityLabel": {
             "audioOnly": "Toggle audio only",
             "audioRoute": "Select the sound device",
-            "callQuality": "Manage call quality",
+            "callQuality": "Manage video quality",
```

Comment:
Torture tests might be failing on this line right here, which is the accessibility label.